src/_lib/modules/resize-observer/resize-observer.directive.ts

```diff
@@ -40,6 +45,10 @@ export class ResizeObserverDirective implements OnInit, OnDestroy {
          * if, for example, the width or height of the element is being animated
          */
         this._nwResize.asObservable().pipe(
+            /**
+             * Store the contentRect of the element so that we can compare changes
+             */
+            tap(entry => this._currentElementRect = entry.contentRect),
```

Comment:
I have it here so that `this._currentElementRect` is updated immediately after we compare successfully compare the resized element [here](https://github.com/NewsWhip/style-guide/blob/4a09caeedb63dfa4d1755849053c52961b766392/src/_lib/modules/resize-observer/resize-observer.directive.ts#L65-L70). I think if have this after the debounce we could end up getting multiple unnecessary calls to `this._nwResize.next` on line 69 because we'll be comparing against an out-of-date element